commit d3fdf43c075f64610244ba93f2af6a1800b81ce3
Author: Volker Krause <vkrause@kde.org>
Date:   Sun Sep 3 19:15:18 2017 +0200

    Fix build without KDocTools
    
    Reviewers: #frameworks
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D7680

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b6932baa8..e1760a1c4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -160,7 +160,9 @@ endif()
 add_definitions(-DTRANSLATION_DOMAIN=\"libplasma5\")
 if (IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
     ki18n_install(po)
-    kdoctools_install(po)
+    if (KF5DocTools_FOUND)
+        kdoctools_install(po)
+    endif()
 endif()
 add_subdirectory(src)
 
